libs/static/src/chains.ts

```diff
@@ -93,13 +93,23 @@ export const chainList = (): Option[] => {
   }))
 }
 
-// DEV: note that ED is double the amount of on-chain ED to prevent weird edge cases of XCM
-export const existentialDeposit: Record<Prefix, number> = {
+// DEV: note that ED is different from the on-chain ED to prevent weird edge cases of XCM
+export const teleportExistentialDeposit: Record<Prefix, number> = {
   ksm: 666666666,
   rmrk: 666666666,
   ahk: 666666666,
   dot: 15000000000,
   ahp: 5000000000,
+  imx: 0,
+  base: 0,
+}
+
+export const existentialDeposit: Record<Prefix, number> = {
+  ksm: 333333333,
+  rmrk: 333333333,
+  ahk: 333333333,
+  dot: 10000000000,
+  ahp: 1000000000,
```

Comment:
![Screenshot 2024-04-03 at 11 42 48](https://github.com/kodadot/nft-gallery/assets/22471030/5415cad5-2265-4c74-b244-34fd31131361)

Reply:
hmm it's  0.1

![CleanShot 2024-04-04 at 16 42 42@2x](https://github.com/kodadot/nft-gallery/assets/44554284/eb6f7555-e8da-4917-9ee4-177869a4d98f)

Reply:
now you two got me thinking, is it the true polkadot assethub existential deposit 0.1 or 0.01? 
i can find both infos on polkadot docs

Reply:
[This one is the current deposit](https://github.com/kodadot/nft-gallery/pull/9972#discussion_r1551483528)

Reply:
updated 4ab191f